[2023-06-06T18:22:23.366-0300] {processor.py:157} INFO - Started process (PID=113350) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:22:23.367-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:22:23.367-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:22:23.367-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:22:23.370-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:22:23.369-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:22:23.370-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:22:23.370-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:22:23.371-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:22:23.381-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2023-06-06T18:23:10.259-0300] {processor.py:157} INFO - Started process (PID=113432) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:23:10.261-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:23:10.263-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:23:10.263-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:23:10.270-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:23:10.268-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:23:10.270-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:23:10.270-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:23:10.271-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:23:10.287-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-06-06T18:23:58.017-0300] {processor.py:157} INFO - Started process (PID=113630) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:23:58.025-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:23:58.027-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:23:58.026-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:23:58.033-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:23:58.032-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:23:58.034-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:23:58.034-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:23:58.034-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:23:58.054-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2023-06-06T18:24:43.412-0300] {processor.py:157} INFO - Started process (PID=113810) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:24:43.420-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:24:43.421-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:24:43.421-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:24:43.429-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:24:43.427-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:24:43.429-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:24:43.429-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:24:43.430-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:24:43.445-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2023-06-06T18:25:34.202-0300] {processor.py:157} INFO - Started process (PID=113962) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:25:34.210-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:25:34.210-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:25:34.210-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:25:34.213-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:25:34.212-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:25:34.213-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:25:34.213-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:25:34.213-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:25:34.223-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-06-06T18:26:33.004-0300] {processor.py:157} INFO - Started process (PID=114099) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:26:33.012-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:26:33.014-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:26:33.014-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:26:33.021-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:26:33.020-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:26:33.022-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:26:33.021-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:26:33.022-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:26:33.039-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2023-06-06T18:28:42.958-0300] {processor.py:157} INFO - Started process (PID=114393) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:28:42.965-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:28:42.966-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:28:42.966-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:28:42.970-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:28:42.969-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:28:42.971-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:28:42.970-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:28:42.971-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:28:42.986-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-06-06T18:29:29.392-0300] {processor.py:157} INFO - Started process (PID=114480) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:29:29.399-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:29:29.400-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:29:29.400-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:29:29.402-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:29:29.402-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:29:29.402-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:29:29.402-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:29:29.402-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:29:29.413-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-06-06T18:30:14.988-0300] {processor.py:157} INFO - Started process (PID=114571) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:30:14.989-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:30:14.989-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:30:14.989-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:30:14.991-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:30:14.991-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:30:14.992-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:30:14.991-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:30:14.992-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:30:15.000-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2023-06-06T18:30:59.357-0300] {processor.py:157} INFO - Started process (PID=114669) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:30:59.364-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:30:59.365-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:30:59.365-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:30:59.367-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:30:59.367-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:30:59.368-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:30:59.367-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:30:59.368-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:30:59.378-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-06-06T18:31:44.931-0300] {processor.py:157} INFO - Started process (PID=114761) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:31:44.938-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:31:44.938-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:31:44.938-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:31:44.941-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:31:44.940-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:31:44.941-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:31:44.941-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:31:44.941-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:31:44.950-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-06-06T18:32:34.400-0300] {processor.py:157} INFO - Started process (PID=114865) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:32:34.407-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:32:34.407-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:32:34.407-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:32:34.410-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:32:34.409-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:32:34.410-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:32:34.410-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:32:34.410-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:32:34.420-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-06-06T18:33:22.142-0300] {processor.py:157} INFO - Started process (PID=114946) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:33:22.149-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:33:22.150-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:33:22.150-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:33:22.152-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:33:22.152-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:33:22.152-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:33:22.152-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:33:22.152-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:33:22.160-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2023-06-06T18:34:09.774-0300] {processor.py:157} INFO - Started process (PID=115040) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:34:09.782-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:34:09.783-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:34:09.783-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:34:09.786-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:34:09.786-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:34:09.787-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:34:09.786-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:34:09.787-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:34:09.797-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-06-06T18:34:56.541-0300] {processor.py:157} INFO - Started process (PID=115179) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:34:56.549-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:34:56.549-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:34:56.549-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:34:56.552-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:34:56.552-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:34:56.553-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:34:56.552-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:34:56.553-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:34:56.564-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-06-06T18:35:43.388-0300] {processor.py:157} INFO - Started process (PID=115316) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:35:43.396-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:35:43.398-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:35:43.397-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:35:43.404-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:35:43.403-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:35:43.405-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:35:43.405-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:35:43.406-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:35:43.425-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2023-06-06T18:36:30.579-0300] {processor.py:157} INFO - Started process (PID=115466) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:36:30.586-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:36:30.586-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:36:30.586-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:36:30.589-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:36:30.588-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:36:30.589-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:36:30.589-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:36:30.589-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:36:30.599-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-06-06T18:37:19.218-0300] {processor.py:157} INFO - Started process (PID=115601) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:37:19.225-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:37:19.225-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:37:19.225-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:37:19.228-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:37:19.227-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:37:19.228-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:37:19.228-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:37:19.228-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:37:19.237-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-06-06T18:38:12.306-0300] {processor.py:157} INFO - Started process (PID=115729) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:38:12.314-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:38:12.316-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:38:12.316-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:38:12.323-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:38:12.321-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:38:12.323-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:38:12.323-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:38:12.324-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:38:12.345-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2023-06-06T18:39:02.839-0300] {processor.py:157} INFO - Started process (PID=115829) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:39:02.841-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:39:02.842-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:39:02.842-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:39:02.847-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:39:02.846-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:39:02.847-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:39:02.847-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:39:02.848-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:39:02.866-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-06-06T18:39:54.233-0300] {processor.py:157} INFO - Started process (PID=115972) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:39:54.241-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:39:54.241-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:39:54.241-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:39:54.244-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:39:54.243-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:39:54.244-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:39:54.244-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:39:54.244-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:39:54.255-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-06-06T18:40:42.783-0300] {processor.py:157} INFO - Started process (PID=116124) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:40:42.784-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:40:42.784-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:40:42.784-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:40:42.787-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:40:42.786-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:40:42.787-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:40:42.787-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:40:42.787-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:40:42.796-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2023-06-06T18:41:28.221-0300] {processor.py:157} INFO - Started process (PID=116210) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:41:28.228-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:41:28.229-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:41:28.229-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:41:28.232-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:41:28.231-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:41:28.232-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:41:28.232-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:41:28.233-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:41:28.244-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-06-06T18:42:16.096-0300] {processor.py:157} INFO - Started process (PID=116297) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:42:16.103-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:42:16.105-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:42:16.105-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:42:16.113-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:42:16.111-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:42:16.113-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:42:16.113-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:42:16.114-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:42:16.139-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2023-06-06T18:43:00.314-0300] {processor.py:157} INFO - Started process (PID=116391) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:43:00.321-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:43:00.321-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:43:00.321-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:43:00.324-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:43:00.323-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:43:00.324-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:43:00.324-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:43:00.324-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:43:00.335-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-06-06T18:43:45.688-0300] {processor.py:157} INFO - Started process (PID=116491) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:43:45.695-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:43:45.697-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:43:45.697-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:43:45.704-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:43:45.702-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:43:45.704-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:43:45.704-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:43:45.705-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:43:45.727-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2023-06-06T18:44:30.008-0300] {processor.py:157} INFO - Started process (PID=116569) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:44:30.015-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:44:30.017-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:44:30.017-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:44:30.025-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:44:30.023-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:44:30.025-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:44:30.025-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:44:30.026-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:44:30.051-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2023-06-06T18:45:17.709-0300] {processor.py:157} INFO - Started process (PID=116727) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:45:17.716-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:45:17.716-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:45:17.716-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:45:17.719-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:45:17.718-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:45:17.719-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:45:17.719-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:45:17.719-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:45:17.730-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-06-06T18:46:05.638-0300] {processor.py:157} INFO - Started process (PID=116858) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:46:05.646-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:46:05.647-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:46:05.647-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:46:05.655-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:46:05.653-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:46:05.655-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:46:05.655-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:46:05.656-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:46:05.680-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-06-06T18:46:49.220-0300] {processor.py:157} INFO - Started process (PID=116950) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:46:49.222-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:46:49.224-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:46:49.224-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:46:49.231-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:46:49.230-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:46:49.232-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:46:49.232-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:46:49.232-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:46:49.247-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-06-06T18:47:35.475-0300] {processor.py:157} INFO - Started process (PID=117041) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:47:35.484-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:47:35.485-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:47:35.485-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:47:35.488-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:47:35.487-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:47:35.488-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:47:35.488-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:47:35.488-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:47:35.501-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-06-06T18:49:02.762-0300] {processor.py:157} INFO - Started process (PID=117206) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:49:02.763-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:49:02.764-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:49:02.764-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:49:02.766-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:49:02.766-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:49:02.767-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:49:02.767-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:49:02.767-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:49:02.776-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2023-06-06T18:49:48.227-0300] {processor.py:157} INFO - Started process (PID=117361) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:49:48.234-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:49:48.235-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:49:48.235-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:49:48.237-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:49:48.237-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:49:48.237-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:49:48.237-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:49:48.237-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:49:48.248-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-06-06T18:50:32.986-0300] {processor.py:157} INFO - Started process (PID=117456) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:50:32.994-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:50:32.994-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:50:32.994-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:50:32.998-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:50:32.997-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:50:32.998-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:50:32.998-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:50:32.998-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:50:33.011-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-06-06T18:51:21.053-0300] {processor.py:157} INFO - Started process (PID=117600) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:51:21.060-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:51:21.060-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:51:21.060-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:51:21.062-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:51:21.062-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:51:21.063-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:51:21.063-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:51:21.063-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:51:21.073-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-06-06T18:52:07.870-0300] {processor.py:157} INFO - Started process (PID=117697) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:52:07.877-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:52:07.878-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:52:07.877-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:52:07.880-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:52:07.880-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:52:07.880-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:52:07.880-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:52:07.880-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:52:07.889-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-06-06T18:52:58.033-0300] {processor.py:157} INFO - Started process (PID=117783) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:52:58.042-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:52:58.043-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:52:58.043-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:52:58.050-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:52:58.048-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:52:58.050-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:52:58.050-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:52:58.051-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:52:58.070-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2023-06-06T18:53:47.466-0300] {processor.py:157} INFO - Started process (PID=117873) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:53:47.474-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:53:47.476-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:53:47.475-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:53:47.483-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:53:47.481-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:53:47.484-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:53:47.483-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:53:47.484-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:53:47.588-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.125 seconds
[2023-06-06T18:54:38.779-0300] {processor.py:157} INFO - Started process (PID=117981) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:54:38.780-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:54:38.780-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:54:38.780-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:54:38.782-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:54:38.782-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:54:38.782-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:54:38.782-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:54:38.782-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:54:38.790-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2023-06-06T18:55:22.664-0300] {processor.py:157} INFO - Started process (PID=118058) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:55:22.672-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:55:22.674-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:55:22.674-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:55:22.682-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:55:22.680-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:55:22.684-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:55:22.683-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:55:22.684-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:55:22.715-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2023-06-06T18:56:12.976-0300] {processor.py:157} INFO - Started process (PID=118195) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:56:12.984-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:56:12.986-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:56:12.985-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:56:12.993-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:56:12.991-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:56:12.994-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:56:12.993-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:56:12.994-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:56:13.016-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2023-06-06T18:57:00.594-0300] {processor.py:157} INFO - Started process (PID=118353) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:57:00.601-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:57:00.602-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:57:00.602-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:57:00.605-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:57:00.604-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:57:00.605-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:57:00.605-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:57:00.605-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:57:00.614-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-06-06T18:57:49.279-0300] {processor.py:157} INFO - Started process (PID=118444) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:57:49.286-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:57:49.287-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:57:49.287-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:57:49.290-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:57:49.289-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:57:49.290-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:57:49.290-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:57:49.290-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:57:49.301-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-06-06T18:58:33.924-0300] {processor.py:157} INFO - Started process (PID=118529) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:58:33.932-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:58:33.934-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:58:33.934-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:58:33.941-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:58:33.939-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:58:33.941-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:58:33.941-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:58:33.942-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:58:33.960-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-06-06T18:59:20.991-0300] {processor.py:157} INFO - Started process (PID=118613) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:59:20.998-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T18:59:20.999-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:59:20.998-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:59:21.001-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:59:21.001-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T18:59:21.001-0300] {logging_mixin.py:149} INFO - [2023-06-06T18:59:21.001-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T18:59:21.002-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T18:59:21.012-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-06-06T19:00:04.536-0300] {processor.py:157} INFO - Started process (PID=118697) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:00:04.544-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:00:04.544-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:00:04.544-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:00:04.547-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:00:04.546-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:00:04.547-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:00:04.547-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:00:04.547-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:00:04.556-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-06-06T19:00:47.088-0300] {processor.py:157} INFO - Started process (PID=118793) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:00:47.095-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:00:47.097-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:00:47.097-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:00:47.104-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:00:47.103-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:00:47.105-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:00:47.105-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:00:47.106-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:00:47.131-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2023-06-06T19:01:31.741-0300] {processor.py:157} INFO - Started process (PID=118869) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:01:31.749-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:01:31.751-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:01:31.750-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:01:31.758-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:01:31.756-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:01:31.759-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:01:31.758-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:01:31.759-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:01:31.783-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-06-06T19:02:14.076-0300] {processor.py:157} INFO - Started process (PID=118974) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:02:14.085-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:02:14.086-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:02:14.086-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:02:14.094-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:02:14.092-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:02:14.095-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:02:14.094-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:02:14.095-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:02:14.112-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-06-06T19:02:56.090-0300] {processor.py:157} INFO - Started process (PID=119099) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:02:56.098-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:02:56.100-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:02:56.100-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:02:56.107-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:02:56.105-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:02:56.107-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:02:56.107-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:02:56.108-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:02:56.125-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-06-06T19:03:42.663-0300] {processor.py:157} INFO - Started process (PID=119184) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:03:42.670-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:03:42.670-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:03:42.670-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:03:42.673-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:03:42.672-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:03:42.673-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:03:42.673-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:03:42.673-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:03:42.683-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-06-06T19:04:22.583-0300] {processor.py:157} INFO - Started process (PID=119269) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:04:22.591-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:04:22.592-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:04:22.592-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:04:22.600-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:04:22.598-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:04:22.600-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:04:22.600-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:04:22.601-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:04:22.623-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2023-06-06T19:05:05.171-0300] {processor.py:157} INFO - Started process (PID=119354) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:05:05.178-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:05:05.178-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:05:05.178-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:05:05.181-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:05:05.180-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:05:05.181-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:05:05.181-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:05:05.181-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:05:05.190-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-06-06T19:05:48.920-0300] {processor.py:157} INFO - Started process (PID=119454) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:05:48.928-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:05:48.928-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:05:48.928-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:05:48.930-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:05:48.930-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:05:48.931-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:05:48.931-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:05:48.931-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:05:48.940-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-06-06T19:06:34.337-0300] {processor.py:157} INFO - Started process (PID=119541) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:06:34.337-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:06:34.338-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:06:34.338-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:06:34.340-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:06:34.340-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:06:34.340-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:06:34.340-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:06:34.340-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:06:34.349-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2023-06-06T19:07:20.390-0300] {processor.py:157} INFO - Started process (PID=119647) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:07:20.391-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:07:20.393-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:07:20.392-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:07:20.398-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:07:20.397-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:07:20.398-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:07:20.398-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:07:20.399-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:07:20.412-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-06-06T19:08:10.042-0300] {processor.py:157} INFO - Started process (PID=119738) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:08:10.051-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:08:10.052-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:08:10.052-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:08:10.060-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:08:10.058-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:08:10.061-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:08:10.060-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:08:10.061-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:08:10.078-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2023-06-06T19:09:00.928-0300] {processor.py:157} INFO - Started process (PID=119830) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:09:00.936-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:09:00.937-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:09:00.937-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:09:00.942-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:09:00.941-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:09:00.942-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:09:00.942-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:09:00.942-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:09:00.955-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-06-06T19:09:45.041-0300] {processor.py:157} INFO - Started process (PID=119917) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:09:45.048-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:09:45.049-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:09:45.049-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:09:45.052-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:09:45.051-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:09:45.052-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:09:45.052-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:09:45.052-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:09:45.065-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-06-06T19:10:34.712-0300] {processor.py:157} INFO - Started process (PID=120006) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:10:34.720-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:10:34.720-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:10:34.720-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:10:34.724-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:10:34.723-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:10:34.724-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:10:34.724-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:10:34.724-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:10:34.734-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-06-06T19:11:22.720-0300] {processor.py:157} INFO - Started process (PID=120110) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:11:22.727-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:11:22.727-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:11:22.727-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:11:22.730-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:11:22.729-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:11:22.731-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:11:22.731-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:11:22.731-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:11:22.742-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-06-06T19:12:09.552-0300] {processor.py:157} INFO - Started process (PID=120200) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:12:09.559-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:12:09.560-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:12:09.560-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:12:09.562-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:12:09.562-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:12:09.563-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:12:09.563-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:12:09.563-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:12:09.574-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-06-06T19:12:55.445-0300] {processor.py:157} INFO - Started process (PID=120287) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:12:55.453-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:12:55.455-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:12:55.454-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:12:55.462-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:12:55.460-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:12:55.462-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:12:55.462-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:12:55.463-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:12:55.484-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2023-06-06T19:13:38.662-0300] {processor.py:157} INFO - Started process (PID=120386) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:13:38.663-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:13:38.664-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:13:38.663-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:13:38.666-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:13:38.666-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:13:38.667-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:13:38.666-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:13:38.667-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:13:38.676-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2023-06-06T19:14:24.656-0300] {processor.py:157} INFO - Started process (PID=120486) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:14:24.663-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:14:24.664-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:14:24.664-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:14:24.667-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:14:24.666-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:14:24.667-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:14:24.667-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:14:24.667-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:14:24.678-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-06-06T19:15:13.382-0300] {processor.py:157} INFO - Started process (PID=120578) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:15:13.390-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:15:13.390-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:15:13.390-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:15:13.393-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:15:13.392-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:15:13.393-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:15:13.393-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:15:13.393-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:15:13.402-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-06-06T19:16:03.614-0300] {processor.py:157} INFO - Started process (PID=120708) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:16:03.624-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:16:03.626-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:16:03.625-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:16:03.634-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:16:03.632-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:16:03.635-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:16:03.635-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:16:03.635-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:16:03.666-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2023-06-06T19:16:45.788-0300] {processor.py:157} INFO - Started process (PID=120865) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:16:45.790-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:16:45.792-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:16:45.792-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:16:45.799-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:16:45.797-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:16:45.799-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:16:45.799-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:16:45.800-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:16:45.816-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-06-06T19:17:29.930-0300] {processor.py:157} INFO - Started process (PID=120968) to work on /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:17:29.939-0300] {processor.py:826} INFO - Processing file /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-06-06T19:17:29.941-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:17:29.940-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:17:29.948-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:17:29.946-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-06-06T19:17:29.949-0300] {logging_mixin.py:149} INFO - [2023-06-06T19:17:29.948-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-06-06T19:17:29.949-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-06-06T19:17:29.966-0300] {processor.py:179} INFO - Processing /home/obidegain/Workspaces/airflow-demo/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
